examples/mediatek/model_export_scripts/llama.py

```diff
@@ -460,6 +462,7 @@ def main():
                 "eos_token_id_tensor": torch.tensor(tokenizer.eos_token_id),
                 "response_cap": args.response_cap,
             },
+            keep_in_memory=True
```

Comment:
Mind sharing what is keep in memory?

Reply:
Hi @cccclai, keep in memory stores the dataset in RAM instead of caching it. It was a temporary workaround for an OSError I encountered on my end. I have since resolved the issue and will remove this argument in the next commit.